tmt/template.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Optional
+
+import tmt
+import tmt.utils
+from tmt.utils import Path, cached_property
+
+DEFAULT_CUSTOM_TEMPLATES_PATH = tmt.utils.CONFIG_PATH.expanduser() / 'templates'
+DEFAULT_PLAN_NAME = "/plans/default"
+INIT_TEMPLATES = ['mini', 'base', 'full']
+TEMPLATE_KEYS = ['story', 'plan', 'test', 'test_metadata']
+
+TemplatesType = dict[str, dict[str, Path]]
```

Comment:
Please, add a comment describing what's the type is for. Once it's not a simple alias like `Foo = int`, especially with `dict` in a `dict`, explaining the data structure would make it easier for developers to grasp its purpose.

Reply:
Added in 78d7217caecb8c3bd05d92445264de9de4bfaf02.